src/snowflake/snowpark/_internal/open_telemetry.py

```diff
@@ -36,7 +36,10 @@
 def open_telemetry_context_manager(func, dataframe):
 
     # trace when required package is installed
-    if open_telemetry_found:
+    if (
+        open_telemetry_found
+        and os.environ.get("SNOWPARK_SPAN_RECORD", "enable") == "enable"
```

Comment:
Two questions:
Is `os.env.get("abc")` this case insensitive?
Do we use true/false for other env variables or enable/disable?

Reply:
1. yes, it is case sensitive
2. seems like we don't have a tradition for this because it only allow string, but I can change it to "True"/"False"